### PR TITLE
Bug fix for snow initialization in GFS_surface_composites.F90

### PR DESCRIPTION
**Pull request by tanyasmirnova**
1. Bug fix for snow initialization in GFS_surface_composites.F90
2. Set compute_fluxes=.true. in module_sf_mynn.F90

### CODE REVIEW

**Comment by SamuelTrahanNOAA** [comment]
You need to merge @mdtoyNOAA's changes into your branch, and create PRs at the fv3atm and ufs-weather-model level.

**Comment by mdtoyNOAA** [comment]
I thought I had already done this, i.e.:
NOAA-GSL/fv3atm — PR #129
NOAA-GSL/ufs-weather-model — PR #122

**Comment by SamuelTrahanNOAA** [comment]
@mdtoyNOAA Those issues are closed. Tanya's current branch has to be up-to-date with the top of RRFS_dev.

**Comment by SamuelTrahanNOAA** [comment]
This PR's branch must be updated to the top of RRFS_dev now that #137 has been merged. We also need PRs at the fv3atm and ufs-weather-model level.

File: physics/module_sf_mynn.F90 [hunk 1]
```diff
@@ -112,7 +112,7 @@ MODULE module_sf_mynn
                                         !1: check input
                                         !2: everything - heavy I/O
   LOGICAL, PARAMETER :: compute_diag = .false.
-  LOGICAL, PARAMETER :: compute_flux = .false.  !shouldn't need compute 
+  LOGICAL, PARAMETER :: compute_flux = .true.  !shouldn't need compute 
                ! these in FV3. They will be written over anyway.
```
Comment (joeolson42) [inline physics/module_sf_mynn.F90:116]: I think this is OK for now. In the near future, this flag will have to made into a namelist option.

**Review by joeolson42 — APPROVED** [review]

**Comment by SamuelTrahanNOAA** [comment]
Does #138 replace this? If so, please close this PR.

**Comment by SamuelTrahanNOAA** [comment]
I need pull requests in fv3atm and ufs-weather-model before I can test or merge this.

**Pull request closed without merging**